pubg_python/base.py

```diff
@@ -83,6 +83,11 @@ def leaderboards(self, game_mode):
         return QuerySet(self.api_client, self.shard_url.join(
             'leaderboards/{}'.format(game_mode)))
 
+    def weapon_mastery(self, **kwargs):
+        player_id = kwargs.pop('player_id', None)
+        return QuerySet(self.api_client, self.shard_url.join(
+            'players/{}/weapon_mastery'.format(player_id)))
```

Comment:
Isn't player id always required in this case? Maybe you can simply add `player_id` as an arg instead of this optional kwarg. Wdyt?

Reply:
Fixed